content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
+- **Version 1:** The payload SHOULD be encrypted using Waku v1 payload encryption. [26/WAKU-PAYLOAD](/spec/26).
+This provides asymmetric and symmetric encryption. The key agreement is performed out of band. And provides an encrypted signature and padding for some form of unlinkability.
+- **Version 2:** The payload SHOULD be encrypted using [35/WAKU2-NOISE](/spec/35).
+Waku Noise protocol provides symmetric encryption and asymmetric key exchange.
 
-This indicates that payloads MUST be encrypted using [WAKU2-PAYLOAD](/spec/26).
+Any `version` value not included in the list is reserved for future use, and the payload SHOULD be interpreted as unencrypted.
 
-This provides for asymmetric and symmetric encryption.
-Key agreement is out of band.
-It also provides an encrypted signature and padding for some form of unlinkability.
+# Security Considerations
 
-### Version 2
+## Confidentiality, integrity, and authenticity
 
-This indicates that payloads MUST be encoded using [35/WAKU-NOISE](/spec/35).
+The level of confidentiality, integrity, and authenticity of the waku message payload is discretionary. Accordingly, the application layer shall utilize the encryption and signature schemes supported by Waku v2 to meet the application-specific privacy needs.
 
-This provides for symmetric encryption and
-asymmetric key-exchange protocols.
+## Reliability of the `timestamp` attribute
 
-# Differences from Whisper / Waku v1 envelopes
+The waku message `timestamp` attribute is set by the sender. Therefore, because message timestamps aren’t independently verified, this field is prone to exploitation and misuse. It should not solely be relied upon for operations such as message ordering.
 
-In Whisper and Waku v1, an envelope contains the following fields: `expiry, ttl, topic, data, nonce`.
+For example, a malicious actor can arbitrarily set the `timestamp` of a waku message to a high value so that it always shows up as the most recent message in a chat application. Applications using waku messages’ `timestamp` attribute are recommended to use additional methods for more robust message ordering. An example of how to deal with message ordering against adversarial message timestamps can be found in the Status protocol, see [6/PAYLOADS](https://specs.status.im/spec/6#clock-vs-timestamp-and-message-ordering).
 
-Since Waku v2 is using libp2p PubSub, some of these fields can be dropped.
-The previous `topic` field corresponds to `contentTopic`.
-The previous `data` field corresponds to the `payload` field.
+## Reliability of the `ephemeral` attribute
 
-# Security Consideration
+The waku message `ephemeral` attribute is set by the sender. Since there is currently no incentive mechanism for network participants to behave correctly, this field is inherently insecure. A malicious actor can tamper with the value of a waku message’s `ephemeral` attribute, and the receiver would not be able to verify the integrity of the message.
```

Comment:
Decoupling from existing specs makes sense to me (i.e. no direct reference to relay/store specs), but I think an example is necessary to clarify. Could just be "for example, nodes storing historical messages, may ignore messages..." etc.